MAUI/CheckBox/Visual-Customization.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Customization in .NET MAUI CheckBox control | Syncfusion
+description: Learn about Customization support in Syncfusion Essential Studio .NET MAUI CheckBox control, its elements and more.
+platform: Maui
+control: CheckBox
+documentation: ug
+---
+
+# Visual Customization in .NET MAUI CheckBox (SfCheckBox)
+
+## Customizing a shape
+
+The check box shape can be customized using the [`CornerRadius`] property. This property specifies the uniform radius value to the every corner of the check box.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="checkBox" Text="CheckBox" IsChecked="True" CornerRadius="5.0"/>
+    
+{% endhighlight %}
+{% highlight c# %}
+
+    SfCheckBox checkBox = new SfCheckBox();
+    checkBox.Text = "CheckBox";
+    checkBox.IsChecked = true;
+    checkBox.CornerRadius = 5.0f;
+    this.Content = checkBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/Visual-Customization/checkboxradii.png)
+
+## Customizing a state color
+
+The default state colors can be customized using the [`CheckedColor`] and [`UncheckedColor`] properties. The checked or indeterminate state color is updated to the [`CheckedColor`] property value when the state is changed to the checked or indeterminate.The unchecked state color is updated to [`UncheckedColor`] property value when the state is changed to unchecked. 
```

Comment:
The default state colors can be customized using the 'CheckedColor' and 'UncheckedColor' properties. The checked or indeterminate state color is updated to the value of the 'CheckedColor' property when the state is changed to checked or indeterminate. The unchecked state color is updated to the value of the 'UncheckedColor' property when the state is changed to unchecked.